lib/mosaic-utils/src/main/java/org/eclipse/mosaic/lib/util/ProcessLoggingThread.java

```diff
@@ -69,28 +69,28 @@ private void flushLog(InputStream stream) {
                         if (id == null || id.length() == 0) {
                             this.log.error(line);
                         } else {
-                            this.log.error("Process {} : {}", this.id, line);
```

Comment:
I'd revert this change. In the case of SUMO, it is not distinguishable anymore if SUMO itself or the ambassador is logging the message:

![image](https://user-images.githubusercontent.com/2386865/120772849-05fd2180-c521-11eb-91a0-c8d86cf93cd2.png)

Yellow: Ambassador, Green: Sumo

Reply:
For the other cases it is fine, too. It would state:

`INFO  Ns3Output - Process ns3 - blablabla`